src/BaselineOfSpec/BaselineOfSpec.class.st

```diff
@@ -0,0 +1,40 @@
+Class {
+	#name : #BaselineOfSpec,
+	#superclass : #BaselineOf,
+	#category : #BaselineOfSpec
+}
+
+{ #category : #baseline }
+BaselineOfSpec >> baseline: spec [ 
+    
+	<baseline>
+
+	| repository |
+	 
+	repository := self packageRepositoryURL.
+       
+	spec for: #common do: [
+		spec postLoadDoIt: #'postload:package:'.
```

Comment:
I do not see BaselineOfSpec>>postLoad:package: is that normal?

Reply:
Thanks :) I was in rush mode.